app/models/report.rb

```diff
@@ -19,6 +19,7 @@ class Report < ApplicationRecord
   def self.inherited(child)
     child.instance_eval do
       scoped_search :relation => :host,        :on => :name,  :complete_value => true, :rename => :host
+      scoped_search :relation => :host,        :on => :id,    :complete_value => true, :rename => :host_id
```

Comment:
Why not search on the `host_id` column? it would save a potentially painful join with the host table. Also, please make this only explicit.